source/Octopus.Tentacle.Tests/Communications/AutofacServiceFactoryFixture.cs

```diff
@@ -66,12 +68,54 @@ public async Task Resolved_WithSingleSource_CanCreateAsyncServices()
             greeting.Should().Be("Hello Fred!");
         }
 
+        [Test]
+        public void Resolved_WithPerLifetimeScopeSingleSource_EachCreateServiceReturnsSameInstance()
```

Comment:
I noticed that the reason this test works this way is because we hold onto the scope within `AutofacServiceFactory`.

Like we discussed, if we want to make it respect `InstancePerLifetimeScope` and return a new instance, then we have to make `AutofacServiceFactory` create a scope when we call `CreateService`. 

But since all of these are singletons, it's not something we need to concern ourselves with now.

I just commented on this for future travellers 🙂